src/controllers/controller.bar.js

```diff
@@ -1,11 +1,10 @@
 import DatasetController from '../core/core.datasetController';
-import defaults from '../core/core.defaults';
 import {Rectangle} from '../elements/index';
 import {clipArea, unclipArea} from '../helpers/helpers.canvas';
 import {isArray, isNullOrUndef, valueOrDefault} from '../helpers/helpers.core';
 import {_limitValue, sign} from '../helpers/helpers.math';
 
-defaults.set('bar', {
+const defaults = {
```

Comment:
You could save a line/variable by doing `BarController.defaults =` here (and then I'd probably move the id declaration up too, so that it still happens first)

Reply:
I didn't do that so far to avoid that I have move too many lines around. We can still do it later, once we decided on the structure of methods and so on.